scripts/start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
What's the importance of this script?

Reply:
I created to run different command if we are in development or Production....
should I use
```suggestion
CMD ["npm", "start"]
```

Reply:
It was to run different commands depending if we are in production or development. Should I use `CMD ["npm", "start"]`

Reply:
Docker will only be used in development. Yes, use `CMD ["npm", "start"]` instead